subsys/spm/Kconfig

```
@@ -61,6 +61,20 @@ config SPM_SERVICE_RNG
 	  This service allows it to request random numbers from the SPM.
 endif
 
+# FIXME: Work-around for Thingy:91 until tooling issues have been worked out.
+config SPM_ALLOW_NON_SECURE_RESET
+	bool "Allow Non-Secure reset"
+	default y if BOARD_NRF9160_PCA20035NS
```

```suggestion
	default y if BOARD_NRF9160_PCA20035
```
Correct me if I'm wrong, but since this is configuration for the SPM, it is the secure board that is relevant.

---

Your are again correct. Thanks I forgot that the board define changes from each image.

---

Fixed now